go.mod

```diff
@@ -100,6 +100,8 @@ require (
 	github.com/godbus/dbus v4.1.0+incompatible
 	github.com/gogo/googleapis v1.3.2 // indirect
 	github.com/gogo/protobuf v1.3.1
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
```

Comment:
sanity check: do we have a cache library in the agent already? I'm surprised this is the first time we've needed to use a caching library.

Reply:
I couldn't find anything. I was surprised by that too.